### ISSUE TITLE
Fix false positives for trailing comment in media-query-list-comma-newline-after

### ISSUE THREAD

**Issue by ota-meshi** [stylelint/stylelint#3644]
> Clearly describe the bug

`media-query-list-comma-newline-after` does not allow newline after comment.

> Which rule, if any, is the bug related to?

`media-query-list-comma-newline-after`

> What CSS is needed to reproduce the bug?

e.g.

```css
@media screen and (color),/* comment */
projection and (color) {

}
```

> What stylelint configuration is needed to reproduce the bug?

e.g.

```json
{
  "rules": {
    "media-query-list-comma-newline-after": "always"
  }
}
```

> Which version of stylelint are you using?

`9.5.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

`npm run test` in this repository.

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

css standard

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors did you get)?

The following warnings were flagged:

```shell
Expected newline after "," (media-query-list-comma-newline-after)
```

---

I added a testcase for PR.
https://github.com/stylelint/stylelint/pull/3643/files#diff-6344cca02f92d2f2edd8aa78014141b9R132
I expected that not be warned, but it was warned.

For the same reason as for the following opinion, I think that should allow a newline after comment.
https://github.com/stylelint/stylelint/pull/3421#discussion_r200484936

What do you think?

**Comment by jeddy3** [comment]
I believe allowing a comment before the newline will make this rule consistent with other similar rules.
